pkg/kafka/client.go

```diff
@@ -0,0 +1,48 @@
+package kafka
+
+import (
+	"crypto/tls"
+	"fmt"
+
+	"github.com/Shopify/sarama"
+)
+
+type Client struct {
+	RecvQ         chan sarama.ConsumerMessage
+	Producer      sarama.SyncProducer
+	ProducerTopic string
+	SignatureKey  string
+}
+
+func tlsConfig(t TLS) *tls.Config {
+	return &tls.Config{
+		InsecureSkipVerify: t.Insecure,
```

Comment:
må vi❓

Reply:
Nope, we really don't. Shamelessly copied this from navikt/deployment. I've cleaned up things a bit in my latest [commit](https://github.com/nais/naiserator/pull/93/commits/4cb9d2a421ace8153ba6dd9bfc9e8bb07a235433).